chalicelib/checks/helpers/cgap_utils.py

```diff
@@ -55,6 +55,10 @@
         "run_time": 12,
         "accepted_versions": ["v9"]
     },
+    "workflow_granite-mpileupCounts": {
+        "run_time": 12,
+        "accepted_versions": ["v13", "v14"]
+    },
```

Comment:
mpileupcounts v13 and v14 are different and we should only accept v14 (v13 uses different default parameters)

Reply:
It can be either v14 or v15 (v14 and v15 are the same)